daprdocs/content/en/reference/api/configuration_api.md

```diff
@@ -0,0 +1,243 @@
+---
+type: docs
+title: "Configuration API reference"
+linkTitle: "Configuration API"
+description: "Detailed documentation on the configuration API"
+weight: 700
+---
+
+## Component file
```

Comment:
You can remove this component file section. It is not needed here in the API spec.

Reply:
Done